modules/curl/8.8.0.bcr.1/patches/add_build_file.patch

```diff
@@ -0,0 +1,334 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 0000000..ff61c6f
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,328 @@
++# Copyright The OpenTelemetry Authors
++# SPDX-License-Identifier: Apache-2.0
++
++load("@bazel_skylib//rules:common_settings.bzl", "bool_flag", "string_flag")
++load("@bazel_skylib//lib:selects.bzl", "selects")
++
++package(features = ["no_copts_tokenization"])
++
++bool_flag(
++    name = "use_mbedtls",
++    build_setting_default = False,
++)
++
++config_setting(
++    name = "use_mbedtls_setting",
++    flag_values = {":use_mbedtls": "true"},
++)
++
++config_setting(
++    name = "not_use_mbedtls_setting",
++    flag_values = {":use_mbedtls": "false"},
++)
++
++selects.config_setting_group(
++    name = "use_boringssl",
++    match_all = [":not_use_mbedtls_setting", "@platforms//os:linux"],
```

Comment:
what about macOS here? any reason to not use this for that as well?

Reply:
On macOS you want to use Secure Transport.  BoringSSL and OpenSSL will not use the native trust store.  I can expand this PR to include that if you want.

Reply:
would be nice if it's easy, but it doesn't need to block this

Reply:
Done.